pkg/kubectl/cmd/rollingupdate.go

```diff
@@ -257,14 +255,13 @@ func RunRollingUpdate(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, arg
 		updateCleanupPolicy = kubectl.RenameRollingUpdateCleanupPolicy
 	}
 	config := &kubectl.RollingUpdaterConfig{
-		Out:            out,
-		OldRc:          oldRc,
-		NewRc:          newRc,
-		UpdatePeriod:   period,
-		Interval:       interval,
-		Timeout:        timeout,
-		CleanupPolicy:  updateCleanupPolicy,
-		UpdateAcceptor: kubectl.DefaultUpdateAcceptor,
+		Out:           out,
+		OldRc:         oldRc,
+		NewRc:         newRc,
+		UpdatePeriod:  period,
+		Interval:      interval,
+		Timeout:       timeout,
+		CleanupPolicy: updateCleanupPolicy,
```

Comment:
that's fine

On Thu, Jul 30, 2015 at 3:26 PM, Dan Mace <notifications@github.com> wrote:

> In pkg/kubectl/cmd/rollingupdate.go
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/11942#discussion_r35910110>
> :
>
> > @@ -257,14 +255,13 @@ func RunRollingUpdate(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, arg
> >  		updateCleanupPolicy = kubectl.RenameRollingUpdateCleanupPolicy
> >  	}
> >  	config := &kubectl.RollingUpdaterConfig{
> > -		Out:            out,
> > -		OldRc:          oldRc,
> > -		NewRc:          newRc,
> > -		UpdatePeriod:   period,
> > -		Interval:       interval,
> > -		Timeout:        timeout,
> > -		CleanupPolicy:  updateCleanupPolicy,
> > -		UpdateAcceptor: kubectl.DefaultUpdateAcceptor,
> > +		Out:           out,
> > +		OldRc:         oldRc,
> > +		NewRc:         newRc,
> > +		UpdatePeriod:  period,
> > +		Interval:      interval,
> > +		Timeout:       timeout,
> > +		CleanupPolicy: updateCleanupPolicy,
>
> Actually, just thought of a question around this: how should an
> implementation be selected? A compatible and automatic way would be to use
> the status field when no percentage is at play, and use the poller when a
> percentage is specified.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/11942/files#r35910110>
> .
>



-- 
Clayton Coleman | Lead Engineer, OpenShift